docs/hub/model-cards.md

```diff
@@ -158,6 +158,23 @@ base_model:
 
 The Hub will infer the type of relationship from the current model to the base model (`"adapter", "merge", "quantized", "finetune"`) but you can also set it explicitly if needed: `base_model_relation: quantized` for instance.
 
+### Specifying a new version
+
+If a new version of your model is available in the Hub, you can specify it in a `new_version` field.  
```

Comment:
Maybe also add that it deep link between versions?

Reply:
see https://github.com/huggingface/hub-docs/pull/1430/files#diff-546bf1c4c3037f2ff17191abae9ca1147f0311a60805a2b0febefb0b122fed91R171